vtex-sites/base.store
# feat: graphql-utils

## What's the purpose of this pull request?
This PR adds the graphql-codegen to this repo. More info at: https://github.com/vtex/faststore/pull/998

Also, I have added a doc to the README on how to generate queries

## How to test it?
For testing, clone this repo and try changing fragments and regenerating graphql queries by yourself. Make sure the new data is fetched
